src/azure-cli/azure/cli/command_modules/acr/agentpool.py

```diff
@@ -61,11 +61,15 @@ def acr_agentpool_update(cmd,
     _, resource_group_name = validate_managed_registry(
         cmd, registry_name, resource_group_name)
 
+    AgentPoolUpdateParameters = cmd.get_models('AgentPoolUpdateParameters', operation_group='agent_pools')
+
+    update_parameters = AgentPoolUpdateParameters(count=count)
+
```

Comment:
Are these changes related to how the SDK is being loaded now?

Reply:
[Unified operation parameter style](https://devblogs.microsoft.com/azure-sdk/migrating-python-management-libraries/#unified-operation-parameter-style)

We have two ways to load these parameters, model style and json style. [example](https://github.com/Azure-Samples/azure-samples-python-management/blob/master/playground/azure/createVM/track2.py#L37)